doc/source/image_description/elements.rst

```diff
@@ -434,29 +432,35 @@ spare_part="number":
   of the requested size. The attribute takes a size value
   and allows a unit in MB or GB, e.g 200M. If no unit is given
   the value is considered to be mbytes. A spare partition
-  can only be configured for the disk image types oem and vmx
+  can only be configured for the disk image type oem
 
 spare_part_mountpoint="dir_path":
   Specify mount point for spare partition in the system.
-  Can only be configured for the disk image types oem and vmx
+  Can only be configured for the disk image type oem
 
 spare_part_fs="btrfs|ext2|ext3|ext4|xfs":
   Specify filesystem for spare partition in the system.
-  Can only be configured for the disk image types oem and vmx
+  Can only be configured for the disk image type oem
 
 spare_part_fs_attributes="attribute_list":
   Specify filesystem attributes for the spare partition.
   Attributes can be specified as comma separated list.
   Currently the attributes `no-copy-on-write` and `synchronous-updates`
   are available. Can only be configured for the disk image
-  types oem and vmx
+  type oem
 
 spare_part_is_last="true|false":
   Specify if the spare partition should be the last one in
-  the partition table. Can only be configured for the vmx
-  disk image type. By default the root partition is the last
-  one and the spare partition lives before it. With this
-  attribute that setup can be toggled.
+  the partition table. Can only be configured for the oem
+  type with oem-resize switched off. By default the root
```

Comment:
I think, you should use backticks here to visualize that it not a concept but a value or an element:

```suggestion
  the partition table. Can only be configured for the `oem`
  type with `oem-resize` switched off. By default the root
```